Makefile

```diff
@@ -261,10 +261,12 @@ clean: clean-certs clean-containers clean-slice ## Stops all containers and remo
 	$(RM) *.box
 
 test: ## executes the test script on a master
+	# This command is broken, as per
```

Comment:
Is there a way to configure it while using `make test` here?
If not, what would you like to be done?

That configuration seems to be incomplete, which is what the linked issue refers to. In particular, the issue refers to `DCOS_LOGIN_UNAME`. That doesn't seem to be in the README or in `test_env.export`.